docs/content/envoy-performance.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Performance
+kind: envoy
+weight: 100
+---
+
+This page provides some performance benchmarks that give an idea of the overhead of using the OPA-Envoy plugin.
+
+### Test Setup
+
+The setup uses the same example Go application that's described in the [Quick Start](#quick-start) section above. Below
+are some more details about the setup:
+
+* Platform: Minikube
+* Kubernetes Version: 1.18.6
+* Envoy Version: 1.10.0
+* OPA-Envoy Version: 0.26.0-envoy
+
+### Benchmarks
+
+The benchmark result below provides the percentile distribution of the latency observed by sending *100 requests/sec*
+to the sample application. Each request makes a `GET` call to the `/people` endpoint exposed by the application.
+
+The graph shows the latency distribution when the load test is performed under the following conditions:
+
+1. **App Only**
+
+In this case, the graph documents the latency distribution observed when requests are
+sent directly to the application ie. no Envoy and OPA in the request path. This scenario is depicted by the
+`blue` curve.
+
+2. **App and Envoy**
```

Comment:
Something funny is going on when this is rendered: 

![image](https://user-images.githubusercontent.com/870638/108361725-2368e280-71f3-11eb-98e9-4c3c033edf35.png)

(All items have a `1.`)

Reply:
Udpated